omnisharp-plugin/pom.xml

```
@@ -16,11 +16,11 @@
   <description>Code Analyzer based on Omnisharp</description>
 
   <properties>
-    <analyzer.version>9.20.0.85982</analyzer.version>
+    <analyzer.version>9.23.2.88755</analyzer.version>
     <!-- The following properties should match https://github.com/SonarSource/sonar-dotnet/blob/master/pom.xml#L58 for the given analyzer version -->
-    <sonar.analyzer.commons>2.7.0.1482</sonar.analyzer.commons>
-    <sonar.version>10.6.0.2114</sonar.version>
-    <sonar.api.impl.version>10.3.0.82913</sonar.api.impl.version>
+    <sonar.analyzer.commons>2.9.0.2753</sonar.analyzer.commons>
+    <sonar.version>10.7.0.2191</sonar.version>
+    <sonar.api.impl.version>10.5.0.89998</sonar.api.impl.version>
```

The [9.23.2](https://github.com/SonarSource/sonar-dotnet/blob/9.23.2.88755/pom.xml) has the following values:

```suggestion
    <sonar.analyzer.commons>2.8.0.2699</sonar.analyzer.commons>
    <sonar.version>10.6.0.2114</sonar.version>
    <sonar.api.impl.version>10.4.1.88267</sonar.api.impl.version>
```

I think we should stick to it